src/main/java/cz/codecamp/logger/loggers/ImperativeMultiLogger.java

```diff
@@ -0,0 +1,26 @@
+package cz.codecamp.logger.loggers;
+
+import cz.codecamp.logger.LogLevelEnum;
+import cz.codecamp.logger.LoggerInterface;
+
+import java.io.File;
+import java.io.PrintStream;
+import java.util.List;
+import java.util.logging.Logger;
+
+public class ImperativeMultiLogger implements LoggerInterface {
+
+    List<LoggerInterface> listOfLoggers;
+
+    public ImperativeMultiLogger(List<LoggerInterface> listOfLoggers) {
+        this.listOfLoggers = listOfLoggers;
+    }
+
+    @Override
+    public void log(LogLevelEnum level, String message) {
+        for (int i = 0; i < listOfLoggers.size(); i++){
```

Comment:
Pokud už máme `List<T>` instanci, je pro iteraci lepší použít `for (LoggerInterface logger : listOfLoggers)`. Pokud bych měl v `listOfLoggers` instanci [`LinkedList`](https://docs.oracle.com/javase/7/docs/api/java/util/LinkedList.html), tak `size()`/`get()` bude pořád dokola proiterovávat list => velmi pomalé.